README.md

```diff
@@ -954,60 +967,61 @@ Liquidity events are the usual way shareholders in private companies sell their
 However, individuals may sometimes be able to buy or sell stock (“gain liquidity”) in
 private companies.
 
-- 🄳 In a few cases, you may be able to sell private company stock to another private party,
-  such as an accredited investor who wants to invest in the company.
-  Sales between private parties like this are also called **private sales** or **secondary
-  market** transactions, to
-  [distinguish them](https://www.investopedia.com/terms/s/secondarymarket.asp) from **primary
-  market** transactions where companies directly sell to investors.
-- 🔸 While many who hold stock hope or expect they need only find a willing buyer, in
-  practice, secondary sales only work out in a few situations.
-  In some cases, you may have luck selling stock privately to an individual, like a board
-  member or former executive who wishes to increase their ownership.
-  You can find further discussion of this topic
-  [on Quora](https://www.quora.com/How-do-employees-in-startups-sell-stock-in-the-secondary-markets).
-- 🔸 Sales generally require the agreement and cooperation of the company, for both
+- 🄳 A **secondary market transaction** (or **private sale**) occurs when private company stock
+  is sold to another private party.
+  These sales are unusual because the value of a private company’s stock is uncertain, but
```

Comment:
This line now technically is misleading—they are unusual not so much because of uncertainty (lots of transactions where values are uncertain occur in markets all the time) but because of buyers and sellers finding each other and because a company may not approve.